include/deal.II/base/function_signed_distance.h

```diff
@@ -212,6 +213,42 @@ namespace Functions
       const double                  tolerance;
       const unsigned int            max_iter;
     };
+
+
+    /**
+     * Signed-distance level set function of a rectangle
+     *
+     * This function is zero on the rectangle, negative "inside" and positive
+     * in the rest of $\mathbb{R}^{dim}$.
+     *
+     * @ingroup functions
+     */
+    template <int dim>
+    class Rectangle : public Function<dim>
```

Comment:
I already thought about this. I decided to use a consistent naming with the existing `Function::SignedDistance::Sphere`. If we use `Function::SignedDistance::HyperRectangle` here, we should probably change the naming of `Function::SignedDistance::Sphere` to `Function::SignedDistance::HyperSphere`as well. Since this would break backwards compatibility, what do you think we should do to achieve consistent naming?

Reply:
That makes sense. Let's leave the name as it is then.